rust/opendp_tooling/src/lib.rs

```diff
@@ -7,6 +7,12 @@ pub mod proven;
 // logic for generating bindings from the core structures below
 pub mod codegen;
 
+#[derive(Debug)]
+pub struct Deprecation {
+    version: String,
```

Comment:
Minor nit, this could be "since" as well, to match, but not blocking.

Reply:
I don't have a preference either way: Will change.